enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -431,6 +433,11 @@ func mergeDiffSnapshot(ctx context.Context, baseSnapshotPath string, diffSnapsho
 				// 3 is the Linux constant for the SEEK_DATA option to lseek.
 				newOffset, err := gin.Seek(offset, 3)
 				if err != nil {
+					// ENXIO is expected when the offset is within a hole at the end of
+					// the file.
+					if err := errors.Unwrap(err); err == syscall.ENXIO {
```

Comment:
Since we're checking for syscall errors, I think we might as well call `syscall.Seek` directly instead of going through the file wrapper.

Reply:
Done